bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -127,26 +128,26 @@ public String toString() {
     /**
      * Filters
      */
-    public enum Filter implements DisplayableSpType, SequenceableSpType, LoggableSpType, ObsoletableSpType {
+    public enum Filter implements DisplayableSpType, SequenceableSpType, LoggableSpType, ObsoletableSpType, IctdType {
 
-        OPEN("Open", "Open", new Some<>(1.6)) {
+        OPEN("Open", "Open", new Some<>(1.6),                     Ictd.unavailable()) {
```

Comment:
I wonder if this will bring into the seqexec, all the dependencies. I think scalaz is not a problem but would doobie and shapeless come along to the seqexec?

Reply:
No, the `doobie` and `shapeless` stuff are not in `pot` so they shouldn't get pulled in.

Reply:
`pot` is certainly a seqexec dependency

Reply:
Doobie and shapeless are already used in the ocs codebase, in the `edu.gemini.util.security` bundle.  I'm using the same versions in `edu.gemini.ictd`, which is not referenced from `edu.gemini.pot`. All that is in `pot` is a description of how to lookup corresponding values in the database -- basically a listing of the associated `String` keys per `enum` value.

It does use a scalaz type, `\&/`.  Do you think that will be problematic for the seqexec?

Reply:
`scalaz` is not an issue as we are moving away from it. It will remain a runtime dependency but that's fine. `shapeless` and `doobie` are not directly used at the moment in `ocs3` so it is not an issue but if we were to add it could cause trouble. I've checked and at the moment we get `shapeless` from `monocle`. We don't import `edu.gemini.util.security`

Reply:
It seems then it shouldn't be a problem if we don't import `ictd` right. Do you get `shapeless` or `doobie` packages on `pot`'s osgi manifest?

Reply:
No, neither appears in the manifest.

Reply:
I think that solves my worry